web/src/pm_list.ts

```diff
@@ -199,16 +205,35 @@ function zoom_in(): void {
     $(".direct-messages-container").removeClass("zoom-out").addClass("zoom-in");
     $("#streams_list").hide();
     $(".left-sidebar .right-sidebar-items").hide();
+    $("#left_sidebar_scroll_container .simplebar-content-wrapper").attr("tabindex", "-1");
+
```

Comment:
This workaround will be removed on solving #30403 which I'll work on soon, but you can ignore this now and proceed for the rest of the review.

Reply:
That's merged, so you can drop this now :tada:.

Reply:
Removed [here](https://github.com/zulip/zulip/compare/d1a4411d73e99adee5ee0a995c9b750d8596ed30..dcf47ef6cdd300ba60505e840a25c16310160f86)